actiontext/app/helpers/action_text/content_helper.rb

```diff
@@ -18,17 +18,31 @@ def sanitize_action_text_content(content)
       sanitizer.sanitize(content.to_html, tags: allowed_tags, attributes: allowed_attributes, scrubber: scrubber).html_safe
     end
 
+    def render_action_text_attachment(attachment, locals: {})
```

Comment:
@georgeclaghorn I've extracted the repeated logic of constructing the `render` call for an Attachable/Attachment.

Is a View Helper the correct refactoring? It felt appropriate, but it also increases the surface area of the public API.  Does marking it as `private` have an impact when mixed into a host Application's helper module?

What alternatives do we have at our disposal?

Reply:
A helper method does feel appropriate in this case. :+1:

> Does marking it as `private` have an impact when mixed into a host Application's helper module?

It doesn’t. All other helpers will be able to call it.

Our policy is that Rails’s public API is whatever’s in [the API docs](https://api.rubyonrails.org). Methods that aren’t in the docs are internal, even if they happen to be publicly-callable for whatever reason.

You can mark this method as `private` or `#:nodoc` it to exclude it from the public API. I don’t feel strongly either way.

Reply:
Thanks for the tip. I've pushed up changes to re-order the methods and to mark `render_action_text_attachment` as `:nodoc:`.